pywebostv/controls.py

```diff
@@ -251,8 +265,12 @@ class SystemControl(WebOSControlBase):
         "notify": {
             "uri": "ssap://system.notifications/createToast",
             "args": [str],
-            "payload": {"message": arguments(0)}
-        }
+            "payload": {
+                "iconData": arguments(1, postprocess=lambda url: get_icon_data(url), default=None) ,
+                "iconExtension": "png",
```

Comment:
This could potentially be turned into:

```python
arguments("icon_ext", default=None)
```

.. We will have to document the supported extensions in the README.md though.

Reply:
Done.